src/runtime_src/core/common/query_requests.h

```diff
@@ -66,9 +66,14 @@ enum class key_type
   rom_time_since_epoch,
 
   xclbin_uuid,
+  group_topology,
```

Comment:
Let's be constistent and name this group_topology_raw and move it down by the other xclbin section query requests.